lisa/analysis/load_tracking.py

```diff
@@ -109,13 +123,26 @@ def df_tasks_signals(self):
 
           * A ``util`` column (the average utilization of a task at time t)
           * A ``load`` column (the average load of a task at time t)
+          * A ``util_est_enqueued`` column (the utilest of a task at time t)
```

Comment:
Since we already use shorter names (e.g. `util` instead of `util_avg`) perhaps we can also `s/util_est_enqueued/enqueued/`

... and add also a column for `ewma` reporesenting `util_est_ewma`?

Reply:
I'm not quite sure about the `enqueued` rename, as it does not even reflect anymore the kind of thing the signal is (utilization). Maybe `enqueued_util` ?

I'll add `util_est_ewma` as well

Reply:
Actually there is nothing more to do to support `util_est_ewma`, since it's just a field of the util est event, so it works out of the box.